### PR TITLE
[KOGITO-9316] - add arm64 library as default

### PR DESCRIPTION
**Pull request by spolti**

### CODE REVIEW

File: docs/modules/ROOT/pages/includes/quarkus-embedded-postgresql.adoc [hunk 1]
```diff
@@ -25,4 +25,20 @@ endif::add-copy-button-to-env-var[]
 --|string 
 |
 
+
+a| [[quarkus-embedded-postgresql_quarkus.embedded.postgresql.startup.wait]]`link:#quarkus-embedded-postgresql_quarkus.embedded.postgresql.startup.wait[quarkus.embedded.postgresql.startup.wait]`
```
Comment (cristianonicolai) [inline docs/modules/ROOT/pages/includes/quarkus-embedded-postgresql.adoc:29]: @spolti is this related?

Comment (fjtirado) [reply docs/modules/ROOT/pages/includes/quarkus-embedded-postgresql.adoc:29]: No, but it was a missing piece of doc, and I think is fine to complete it

**Review by fjtirado — APPROVED** [review]
